packages/firestore/lib/src/document_reference.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of firebase_firestore;
+
+/// A DocumentReference refers to a document location in a Firestore database
+/// and can be used to write, read, or listen to the location.
+///
+/// The document at the referenced location may or may not exist.
+/// A DocumentReference can also be used to create a CollectionReference
```

Comment:
Use `[]` for CollectionReference to get a free link in the docs?

Reply:
Done